python/foglamp/services/core/service_registry/service_registry.py

```diff
@@ -87,6 +86,12 @@ def _remove_from_scheduler_records(cls, service_name):
         if service_name in ("FogLAMP Storage", "FogLAMP Core"):
             return
 
+        # Require a local import in order to avoid circular import references
+        from foglamp.services.core import server
```

Comment:
looks a sensible solution, unless we have a solid solution for circular dependency issue!
 https://docs.python.org/3/faq/programming.html#what-are-the-best-practices-for-using-import-in-a-module